### PR TITLE
Homework3

### PR DESCRIPTION
**Pull request by Srgiy**

### CODE REVIEW

File: task3.2.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const sarah = {
+    name: 'Sarah',
+    job: {
+      name: 'Andersen',
+      hiringDate: '23.07.2020'
+    }
+  }
+  
+let {
```
Comment (StanislawVictorovich) [inline task3.2.js]: Please do it in a function

Comment (Srgiy) [reply task3.2.js]: +

Comment (Srgiy) [reply task3.2.js]: i did

File: task3.3.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function delateId(arr) {
+    const unique = new Set()
+    let result = []
+    
+for (let i = 0; i < arr.length; i++){
```
Comment (StanislawVictorovich) [inline task3.3.js:7]: smth wrong with indents. please fix it

**Review by StanislawVictorovich — APPROVED** [review]

**Pull request merged**